src/plugins/auto_batch/src/plugin.cpp

```diff
@@ -172,8 +172,7 @@ std::shared_ptr<ov::ICompiledModel> Plugin::compile_model(const std::shared_ptr<
                 const auto& static_shape = input->get_shape();
                 if (static_shape[0] != 1)
                     OPENVINO_THROW("Auto-batching does not reshape/re-batch originally batched networks!");
-                batched_inputs.insert(
-                    ov::op::util::get_ie_output_name(params[input_id]->output(0)));  // batched dim for the input
+                batched_inputs.insert(params[input_id]->output(0).get_index());  // batched dim for the input
```

Comment:
`get_index()` will return index of this port among all output ports of a node. For `Parameter` node, we have the only output, so `get_index` is always `0`
I suppose `batched_inputs` should contain `input_id`